check_gha_pinning/test_integration.py

```diff
@@ -0,0 +1,91 @@
+import pathlib
+from typing import Iterable
+from unittest.mock import patch
+
+import pytest
+import requests
+from github import Github
+from github.GithubException import GithubException
+from github.Repository import Repository
+from github.Tag import Tag
+
+from check_gha_pinning import check_pinning
+
+
+@pytest.mark.integration
+@patch("os.environ", {"GHA_PINNING_SKIP_GIT_CHECK": "1"})
+def test_integration_no_git_remote(tmp_path):
```

Comment:
```suggestion
def test_integration_no_git_remote(tmp_path: py.path.local) -> None:
```
from memory, please check

Reply:
`pathlib.Path` works for typing, so using that!